gpu-operator-must-gather.sh

```
@@ -0,0 +1,256 @@
+echo "#"
+echo "# nvidia-bug-report.sh"
+echo "#"
+echo ""
+
+for pod in $($K get pods -lopenshift.driver-toolkit -oname -n $OPERATOR_NAMESPACE; $K get pods -lapp=nvidia-driver-daemonset -oname -n $OPERATOR_NAMESPACE; $K get pods -lapp=nvidia-vgpu-manager-daemonset -oname -n $OPERATOR_NAMESPACE);
+do
+    pod_nodename=$($K get $pod -ojsonpath={.spec.nodeName} -n $OPERATOR_NAMESPACE)
+    echo "Saving nvidia-bug-report from ${pod_nodename} ..."
+
+    $K exec -n $OPERATOR_NAMESPACE $pod -- bash -c 'cd /tmp && nvidia-bug-report.sh' >&2 || \
+        (echo "Failed to collect nvidia-bug-report from ${pod_nodename}" && continue)
```

Shellcheck (0.10.0)

[warning] This only exits the subshell caused by the (..) group.

(SC2106)